metron-platform/metron-common/src/test/java/org/apache/metron/common/field/transformation/StellarTransformationTest.java

```diff
@@ -28,39 +28,16 @@
 import org.apache.metron.stellar.common.CachingStellarProcessor;
 import org.apache.metron.stellar.dsl.Context;
 import org.json.simple.JSONObject;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import java.util.Arrays;
-import java.util.Collection;
 import java.util.HashMap;
+import java.util.stream.Stream;
 
-@RunWith(Parameterized.class)
-public class StellarTransformationTest {
-  Context context;
-  public StellarTransformationTest(Cache<CachingStellarProcessor.Key, Object> cache) {
-    if(cache == null) {
-      context = Context.EMPTY_CONTEXT();
-    }
-    else {
-      context = new Context.Builder().with(Context.Capabilities.CACHE, () -> cache).build();
-    }
-  }
-
-  @Parameterized.Parameters
```

Comment:
Won't removing this change the types of validations this suite of tests will perform?

Reply:
It would, if data had actually been used by the tests. Since it wasn't, it got dropped. Other parameterized tests use the newer setup.